docs/index.rst

```diff
@@ -6,23 +6,44 @@
 Welcome to microraiden's documentation!
 =======================================
 
+What is µRaiden?
+-----------------
+µRaiden (read: Micro Raiden) is a payment channel framework for frequent, fast and free ERC20 token based micropayments between two parties. 
+It comes as a set of open source libraries, documentation, and code examples for multiple use cases, ready to implement on the Ethereum mainnet. 
+Whereas its big brother, the Raiden Network, aims to allow for multihop transfers via a network of bidirectional payment channels, µRaiden already enables to make micropayments through unidirectional payment channels.
 
-Contents
-=========
+.. figure:: /diagrams/uRaidenOverview.png
+   :alt:
+
+Try out the demos
+------------------
+
+We have deployed some demo applications that make extensive use of µRaiden in your browser.
+Although you need some testnet-Ether and MetaMask, it's a very easy starting point 
+to try out µRaiden.
+Just follow the instructions on  
+https://demo.micro.raiden.network/
+
+Next steps
+-----------
+If you want to start playing with microraiden, a good starting point is to check out the :doc:`Tutorials <tutorials/index>` section.
+Best you start with the :doc:`developer setup <tutorials/index>` and try our :doc:`Blockchain Tutorial <tutorials/index>` afterwards.
+
+Documentation Content
+----------------------
 
 .. toctree::
    :maxdepth: 2 
-   :caption: Contents:
 
-   proxy-tutorial
-   rest-api
-   dev-overview
-   jsclient/index
+   introduction/index
+   specifications/index
+   applications/index
```

Comment:
Applications is a bit confusing. It makes me think of applications built on top of uRaiden. 
How about `uRaiden Components` or `uRaiden Packages`, `uRaiden Installable Components`?
This should be right after the `Introduction`.

I would have actually thought of something like this (no Applications folder, which makes it harder to understand the docs imho):
```
introduction
tutorials
pyserver
pyclient
jsclient
contract
```

Reply:
we had this before and can roll it back

Reply:
I like bundling pyserver, pyclient and jsclient. They kind of belong together.